app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  validates :name, :surname, :city, presence: true
+  validates :name, :surname, format: /[A-Z][a-z]*/
```

Comment:
from regexp you allow for empty name and surename, I think you want + in place of *. 

ps If user name 2 names or number in name or other strange stuff?, just present should be enough